api/src/main/java/uk/gov/hmcts/PaymentApiApplication.java

```diff
@@ -58,7 +68,10 @@ public CacheManager cacheManager() {
                     .build()),
             new CaffeineCache("sites", Caffeine.newBuilder()
                 .expireAfterWrite(48, TimeUnit.HOURS)
-                .build())
+                .build()),
+        new CaffeineCache("userInfoCache", Caffeine.newBuilder()
```

Comment:
@AnilDhawaleGitHub Did we have a requirement on Caching?

Reply:
no we have removed it